<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LMMUM-LP-0957" failures="1" tests="8" name="My Loan" time="0.19" errors="0" timestamp="2024-12-27T09:20:23 IST">
  <testcase classname="TestNGTutorial.annotation1" name="Test1" time="0.021"/>
  <testcase classname="TestNGTutorial.annotation1" name="Test3" time="0.002"/>
  <testcase classname="TestNGTutorial.annotation1" name="Test3" time="0.001"/>
  <testcase classname="TestNGTutorial.annotation1" name="Test4" time="0.003"/>
  <testcase classname="TestNGTutorial.annotation2" name="Test5" time="0.016">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNGTutorial.annotation2.Test5(annotation2.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- Test5 -->
  <testcase classname="TestNGTutorial.annotation2" name="Test6" time="0.003"/>
  <testcase classname="TestNGTutorial.annotation2" name="Test7" time="0.003"/>
  <testcase classname="TestNGTutorial.annotation2" name="Test8" time="0.003"/>
</testsuite> <!-- My Loan -->
